builtin/logical/pki/path_acme_account.go

```diff
@@ -43,6 +43,13 @@ func addFieldsForACMEPath(fields map[string]*framework.FieldSchema, pattern stri
 			Required:    true,
 		}
 	}
+	if strings.Contains(pattern, framework.GenericNameRegex("policy")) {
+		fields["policy"] = &framework.FieldSchema{
+			Type:        framework.TypeString,
+			Description: `The policy name to pass through to the CIEPS service`,
+			Required:    true,
```

Comment:
I'm worried about this being required in OSS

Reply:
Its only listed if we call the function with a path that contains the special framework value of `policy`. At least for now that doesn't happen, so it should be okay.

Or have you spotted something I'm overlooking?